specs/networking/libp2p-standardization.md

```diff
@@ -0,0 +1,192 @@
+ETH 2.0 Networking Spec - Libp2p standard protocols
+===
+
+# Abstract
+
+Ethereum 2.0 clients plan to use the libp2p protocol networking stack for
+mainnet release. This document aims to standardize the libp2p client protocols,
+configuration and messaging formats.
+
+# Libp2p Protocols
+
+## Transport
+
+This section details the libp2p transport layer that underlies the
+[protocols](#protocols) that are listed in this document.
+
+Libp2p allows composition of multiple transports. Eth2.0 clients should support
+TCP/IP and optionally websockets. Websockets are useful for implementations
+running in the browser and therefore native clients would ideally support these implementations 
+by supporting websockets.
+
+An ideal libp2p transport would therefore be TCP/IP with a fallback to
```

Comment:
In libp2p there's no concept of a "fallback" transport. All transports are equal, and each advertises a 1..n multiaddrs. It's up to the initiator to decide which transport they'll dial.

Reply:
Sure. I was referring to this:
https://github.com/libp2p/rust-libp2p/blob/master/core/src/transport/choice.rs#L62-L76
Which, given a multiaddr tries to dial the initial transport, if that fails, then it dials the second transport. By fail, the Mutliaddr may not support the first transport. 
I agree this is not really a fallback, it's selecting the transport associated with the multiaddr. 
I will re-word this.